include/onnxruntime/core/framework/tensor_shape.h

```diff
@@ -10,13 +10,43 @@
 #include <gsl/gsl>
 #include "onnxruntime_config.h"
 
+#ifdef _MSC_VER
+#pragma warning(push)
+#pragma warning(disable : 4127)
+#endif
+
+#include <absl/container/inlined_vector.h>
```

Comment:
If a warning is found in external code, we don't treat it as error.  So is this ifdef still needed?

Reply:
Yes, because it was failing one or more of the CI builds.